runtime/common/src/lib.rs

```diff
@@ -3,3 +3,33 @@
 pub mod constants;
 pub mod extensions;
 pub mod weights;
+
+/// Macro to set a value (e.g. when using the `parameter_types` macro) to either a production value
+/// or to an environment variable or testing value (in case the `frequency-rococo-local` feature is selected or in instant-sealing mode).
+/// Note that the environment variable is evaluated _at compile time_.
+///
+/// Usage:
+/// ```Rust
+/// parameter_types! {
+/// 	// Note that the env variable version parameter cannot be const.
+/// 	pub LaunchPeriod: BlockNumber = prod_or_local_or_env!(7 * DAYS, 1 * MINUTES, "FRQCY_LAUNCH_PERIOD");
+/// 	pub const VotingPeriod: BlockNumber = prod_or_local_or_env!(7 * DAYS, 1 * MINUTES);
```

Comment:
Watch out. This `const` MUST be removed to use the env var version.

Reply:
That is just a watch out on how to use it. No issue, just how to use the macro. It is there in the documentation, but wanted to highlight it.